src/common/components/IconBox/RoundIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, ChakraProps } from '@chakra-ui/react';
+import React from 'react';
+
+interface RoundIconProps extends ChakraProps {
+  icon: React.ReactNode;
+}
+
+export const RoundIcon = ({ icon = null, ...props }: RoundIconProps) => {
+  return (
+    <Box
```

Comment:
For the border gradient, i tried using `_before` method from this [article](https://css-tricks.com/gradient-borders-in-css/) but it doesn't work due to z index, for some reason, the parent container needs to define z-index: 1 for the border to show. Having 2 divs isn't ideal because we'd have to pass props to two divs. Here's the code I tried:
```
export const IconBox2 = ({
  children,
  ...props
}: PropsWithChildren<IconBoxProps>) => {
  return (
    <Box
      position="relative"
      boxSizing="border-box"
      color="#FFF"
      background="#000"
      backgroundClip="padding-box"
      border="solid 2px transparent"
      borderRadius="1em"
      _before={{
        content: "''",
        position: 'absolute',
        top: 0,
        right: 0,
        bottom: 0,
        left: 0,
        zIndex: -1,
        margin: '-5px',
        borderRadius: 'inherit',
        background: 'linear-gradient(to right, red, orange)',
      }}
      {...props}
    >
      {children}
    </Box>
  );
};
```
If you have any suggestions, that would be great!

Reply:
```
  <Box
      display="flex"
      alignItems="center"
      justifyContent="center"
      w={{ base: "16", md: "20" }}
      h={{ base: "16", md: "20" }}
      borderRadius="50%"
      position="relative"
      padding="0.5"
      bgColor="blue.200"
      _before={{
        content: "''",
        position: "absolute",
        inset: "-1px", // border thickness
        zIndex: -1,
        borderRadius: "inherit", // since border-radius is 50% (full) so inherit won't create any issue
        bg: "linear-gradient(45deg, #ff4262, #ffd70e, #ff38ac)",
      }}
      {...props}
    >
      {icon}
    </Box>
    ```
    @tzeweiwee do you think this is better?

Reply:
yeah i tried something like that but it doesn't work
<img width="131" alt="image" src="https://user-images.githubusercontent.com/19740800/208039196-eb90121b-751c-411e-949e-3da42460c929.png">

i'll explore again, but one thing i discovered is if i add `z-index:1` to Page's Box component, it will show
![image](https://user-images.githubusercontent.com/19740800/208039443-04c197dc-8f35-4513-aed4-ea772a7a16d0.png)
<img width="140" alt="image" src="https://user-images.githubusercontent.com/19740800/208039487-935cd4c5-5d60-4b9d-bde8-0579f2e8446d.png">

Reply:
This is working on my end: 🤔 

![CleanShot 2022-12-16 at 12 37 10](https://user-images.githubusercontent.com/53527664/208039920-b7f9c0ca-c041-4c63-8c47-2d5db11f3218.png)

Reply:
I've updated the code with your neat trick, thanks @sushanyadav 🙏